package.json

```diff
@@ -42,12 +41,14 @@
     "test": "codeceptjs run-multiple parallel",
     "test:functional": "NODE_TLS_REJECT_UNAUTHORIZED=0 MOCHAWESOME_REPORTFILENAME=functional codeceptjs run-multiple parallel --grep '(?=.*)^(?!.*(@smoke-tests|@flaky|@nightlyOnly))' --reporter mocha-multi --verbose",
     "test:functional-nightly": "yarn playwright install chromium && yarn playwright install firefox && yarn playwright install webkit && yarn playwright test --project=chromium --project=webkit --project=firefox",
-    "test:functional-chromium": "yarn playwright install chromium && yarn playwright install firefox && yarn playwright install webkit && yarn playwright test --project=chromium",
-    "test:functional-webkit": "yarn playwright install chromium && yarn playwright install firefox && yarn playwright install webkit && yarn playwright test  --project=webkit",
-    "test:functional-firefox": "yarn playwright install chromium && yarn playwright install firefox && yarn playwright install webkit && yarn playwright test --project=firefox",
+    "test:functional-chromium": "yarn playwright install chromium  && yarn playwright test --project=chromium",
+    "test:functional-webkit": "yarn playwright install webkit && yarn playwright test  --project=webkit",
+    "test:functional-firefox": "yarn playwright install firefox  && yarn playwright test --project=firefox",
+    "test:functional-ipadPro11": " yarn playwright install webkit  && yarn playwright test --project=ipadPro11 ",
+    "test:functional-galaxyS4": "  yarn playwright install chromium  && yarn playwright test --project=GalaxyS4 ",
     "test:functional-nightly-preview": "yarn playwright install chromium && yarn playwright test --project=preview",
     "test:smoke": "yarn npm audit --recursive --environment production && MOCHAWESOME_REPORTFILENAME=smoke REPORT_DIR=test-results/smokeTest REPORT_FILE=test-results/smokeTest/results.xml codeceptjs run --grep '@smoke-tests' --reporter mocha-multi --verbose",
-    "test:local": "yarn playwright test --project=webkit --repeat-each=1",
+    "test:local": "yarn playwright test --project=ipad7 --ui --headed --repeat-each=1",
```

Comment:
should this still be webkit for testing locally?

Reply:
This script used for local testing and keeps changing. Better I will rollback to master.